src/applications/dhp-connected-devices/containers/App.jsx

```diff
@@ -16,6 +17,9 @@ export default function App() {
 
   return (
     <div className="usa-grid-full margin landing-page">
+      <MetaTags>
+        <meta name="robots" content="noindex" />
+      </MetaTags>
```

Comment:
Is there a way to add a test that MetaTags exist?

Reply:
We thought about that but it seems like all that would do is test the library. We're importing `MetaTags` from `react-meta-tags` library. Should we still run a test for that?

Reply:
I think we can still test that the MetaTags exists on the page without testing its functionality. So if someone accidentally removes it, we have a breaking test that will let them know something isn't right

Reply:
We gave up and created a card DHP-939 to write a test later